.jshintignore

```diff
@@ -8,5 +8,8 @@ modules/shortcodes/js/jquery.cycle.min.js
 modules/theme-tools/responsive-videos/responsive-videos.min.js
 modules/**/test-*.js
 
+_inc/client/**
+extensions/blocks/**
```

Comment:
Let's just ignore everything in `/extensions` — we'll soon probably add `extensions/plugins` folder and I'm not expecting that folder to ever contain JS that concerns jshint.